projects/project-tooling.md

```diff
@@ -0,0 +1,38 @@
+# Project Tooling SIG
+
+## Description
+
+Currently, a variety of tools exist around OpenTelemetry for various
+organizational purposes -- GitHub bots, for example, or the Stack Overflow to
```

Comment:
Q: if a SIG wants to use a new github integration, would they now come to this SIG, or would they continue coming to the TC? Would this new SIG handle the TC interaction required for this?

Reply:
The eventual goal is that this SIG would provide stuff like this 'as a service' (however we define that) which would automate much of the process. I would like to get humans out of the way as much as possible except for approvals.